modules/modelops/pages/feast-teradata.adoc

```diff
@@ -0,0 +1,253 @@
+
+
+[source, python]
+----
+entity_rows = [
+        {
+            "driver_id": 1001,
+        },
+        {
+            "driver_id": 1002,
+        },
+    ]
+features_to_fetch = [
+            "driver_hourly_stats:acc_rate",
+            "driver_hourly_stats:conv_rate",
+            "driver_hourly_stats:avg_daily_trips"
+        ]
+returned_features = store.get_online_features(
+    features=features_to_fetch,
+    entity_rows=entity_rows,
+).to_dict()
+for key, value in sorted(returned_features.items()):
+    print(key, " : ", value)
+----
+
+== How to set SQL Registry
+
+Another important thing is the SQL Registry. We first make a path variable which uses the username, password, database name etc. to make a connection string which it then uses to establish a connection to Teradata's Database.
+[source, python]
+----
+path = 'teradatasql://'+ teradata_user +':' + teradata_password + '@'+host + '/?database=' + teradata_database + '&LOGMECH=' + teradata_log_mech
+----
+
+It will create the following table in your database:
+
+* Entities (entity_name,project_id,last_updated_timestamp,entity_proto)
+
+* Data_sources (data_source_name,project_id,last_updated_timestamp,data_source_proto)
+
+* Feature_views (feature_view_name,project_id,last_updated_timestamp,materialized_intervals,feature_view_proto,user_metadata)
+
+* Request_feature_views (feature_view_name,project_id,last_updated_timestamp,feature_view_proto,user_metadata)
+
+* Stream_feature_views (feature_view_name,project_id,last_updated_timestamp,feature_view_proto,user_metadata)
+
+* managed_infra (infra_name, project_id, last_updated_timestamp, infra_proto)
+
+* validation_references (validation_reference_name, project_id, last_updated_timestamp, validation_reference_proto)
+
+* saved_datasets (saved_dataset_name, project_id, last_updated_timestamp, saved_dataset_proto)
+
+* feature_services (feature_service_name, project_id, last_updated_timestamp, feature_service_proto)
+
+* on_demand_feature_views (feature_view_name, project_id, last_updated_timestamp, feature_view_proto, user_metadata)
+
+Additionally, if you want to see a complete (but not real-world), end-to-end example workflow example, see the `demo/test_workflow.py` script. This is used for testing the complete feast functionality.
+
+Further reading on the benefits of scalable registry https://docs.feast.dev/tutorials/using-scalable-registry[Feast Scalable Registry]
```

Comment:
We need a summary at the end that summarizes what the article was about. Also, we have a convention for the "Further reading" block. Check out for example: https://quickstarts.teradata.com/run-vantage-express-on-aws.html